test/JDBC/input/BABEL-736.sql

```diff
@@ -0,0 +1,95 @@
+SELECT degrees(10)
```

Comment:
Where is this test running? I don't see it in the jdbc_schedule. Is this intentional?

Reply:
yes it was given in the jira so have added it

Reply:
I mean this test file.. where is this test file running?